pkg/param/param_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/Masterminds/sprig"
 	. "gopkg.in/check.v1"
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Annoying - but `goimports` is adding this - https://github.com/golang/go/issues/28428